projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.html

```diff
@@ -68,7 +65,6 @@
         [tabindex]="getTabIndex(node, depth)"
         (focus)="depth || reinitializeMenu()"
         (keydown.space)="toggleOpen($any($event))"
-        (keydown.esc)="back()"
```

Comment:
Removing this call without an alternative would result in keyboard controls being negatively affected.